pages/glossary/programatically-determined-link-context.md

```diff
@@ -0,0 +1,19 @@
+--
+title: Programmatically Determined Link Context
+key: programmatically-determined-link-context
+--
+
+In HTML, information that is programmatically determinable from a link includes [text nodes][] that are [included in the accessibility tree][] with a non-empty ("") [accessible name][] and are
+
+- in the same paragraph (HTML `p` element) as the link;
+- in the same list (HTML element with the semantic role of `listitem`) as the link;
+- in the same table cell (HTML element with the semantic role of `cell` or `gridcell`) as the link;
+- in a table header cell (HTML element with the semantic role of `columnheader` or `rowheader`) that is [associated][] with the table cell (HTML element with the semantic role of `cell` or `gridcell`) of which the link is a descendent;
+- nodes of elements referenced by an `aria-describedby` attribute of the link element.
```

Comment:
I know it's how the WCAG def is written, but I'm not super happy about dropping headings from the context.

Reading technique ARIA7, I kinda guess that the idea is to explicitly include them through a `aria-describedby` or `aria-labelled-by` attribute when needed.

Maybe it's worth illustrating this by adding examples to the definition (like in #994).

Reply:
Headings should still be included in the programmatically determined context definition. It's just that the programmatically determined link context definition is stricter.